.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,53 +1,28 @@
 ## Description
 
-<!--- 
-Describe the purpose of this PR along with any background information and the impacts of the proposed change. 
-For the benefit of the community, please do not assume prior context.
-
-Provide details that support your chosen implementation, including: 
-- breaking changes
-- alternatives considered
-- changes to the API
-- demos (screenshots, videos) if you find that useful
-- etc.
--->
-
-
-## References
-
-<!--- 
-Include any links supporting this change such as a:
-
-- GitHub Issue/PR number addressed or fixed
-- Auth0 Community post
-- StackOverflow post
-- Support forum thread
-- Related pull requests/issues from other repos
-
-If there are no references, simply delete this section.
--->
+<!-- Describe the purpose of this PR along with any background information and the impacts of the proposed change. 
+For the benefit of the community, please do not assume prior context. -->
 
+## Checklist
 
-## Testing
+**Note:** Checklist required to be completed before a PR is considered to be reviewable.
```

Comment:
I believe this is important piece for us to be able to point to and say "hey, you didn't account for tests or documentation changes".